src/index.ts

```diff
@@ -243,8 +243,9 @@ function removeQuerystring(id: string) {
 
 const isFile = (path?: string | undefined): path is string => {
   try {
-    return !!path && fs.statSync(path).isFile()
+    return (path && fs.statSync(path, { throwIfNoEntry: false })?.isFile()) || false;
```

Comment:
```suggestion
    return!!(path && fs.statSync(path, { throwIfNoEntry: false })?.isFile());
```

Reply:
I've decided to split the logic into 2 lines.

Reply:
They're redundant to have two `return false` like codes IMO.